frontend/benefit/handler/src/components/applicationList/useApplicationList.ts

```diff
@@ -16,7 +17,7 @@ interface ApplicationListProps {
 
 const getFullName = (firstName: string, lastName: string): string => {
   const name = `${firstName || ''} ${lastName || ''}`;
-  return name === ' ' ? '-' : name;
+  return name === ' ' || isEmpty(name) ? '-' : name;
```

Comment:
this returns now always ' ' ?  Perhaps this would work: `[firstName,lastName].join(' ')` Join drops the empty values automatically.

Reply:
good point, fixed